monai/utils/module.py

```diff
@@ -591,7 +591,7 @@ def version_geq(lhs: str, rhs: str) -> bool:
 
     """
     lhs, rhs = str(lhs), str(rhs)
-    pkging, has_ver = optional_import("pkg_resources", name="packaging")
+    pkging, has_ver = optional_import("packaging.Version")
```

Comment:
For this one, did you try it works locally? 
```
>>> from packaging import Version
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'Version' from 'packaging' (/usr/local/lib/python3.10/dist-packages/packaging/__init__.py)
```

Reply:
Yes, we tried and this change works locally on our machines.